test/lib/code_corps_web/views/project_view_test.exs

```diff
@@ -94,14 +94,6 @@ defmodule CodeCorpsWeb.ProjectViewTest do
                 "type" => "task-list"
               }
             ]
-          },
-          "tasks" => %{
-            "data" => [
-              %{
-                "id" => task.id |> Integer.to_string,
-                "type" => "task"
-              }
-            ]
           }
```

Comment:
Did you check how this works with ember. Do we access `project.tasks` on the client end anywhere? I seem to remember this missing causing problems for us, but the problem may have gone away due to introducing task lists.

Reply:
This is breaking things in Ember. Even worse, I wanted to be able to specify the tasks `index` to load only `open` tasks by default, but this appears to break functionality if the tasks are not included on the project.

There's some work to be done here figuring out the best way forward.